charts/pokt-http-txdb/templates/additionalManifests.yaml

```diff
@@ -0,0 +1,7 @@
+{{ range .Values.additionalManifests }}
```

Comment:
Nice, this is a cool way to specify additional manifests rather than having to add additional items under templates/